thumbor/app.py

```diff
@@ -7,24 +7,25 @@
 # Licensed under the MIT license:
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
-import tornado.web
-import tornado.ioloop
 
+import tornado.ioloop
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.ioloop'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)